apps/site/assets/css/_footer.scss

```diff
@@ -58,38 +139,303 @@
 // MOBILE FOOTER STYLES
 //
 
-@include media-breakpoint-down(sm) {
+@include media-breakpoint-up(lg) {
   .m-footer__group {
-    display: none;
+    padding-left: 40px;
+  }
+}
+
+address {
+  .m-footer__item {
+    a {
+      color: $brand-primary;
+    }
+
+    @include media-breakpoint-down(xs) {
+      font-size: 14px;
+      line-height: 21px;
+    }
   }
+}
 
-  .m-footer__group-title {
-    display: none;
+.call-us-container {
+  margin-bottom: -50px;
+  background: $brand-primary-lightest-contrast;
+  padding-bottom: 0;
+  padding-top: 22px;
```

Comment:
This padding-top should increase with larger breakpoints - to 32px (sm, md) then 40px (lg, xl).